docs/index.rst

```diff
@@ -871,6 +918,58 @@ If the project isn't found, a ``404`` response will be given. The response body'
      }]
    }
 
+.. http:get:: /projects/(str:project_id)/channels/
+
+  Retrieves the :ref:`descriptions <data-channels>` of the channels accessible
+  to the project with the id ``project_id``.
+
+  Only accessible if the authenticated user has :ref:`admin permission
+  <permissions-admin>` or has permissions associated with project
+  ``project_id``.
+
+  .. sourcecode:: http
+
+     GET /projects/21/channels/ HTTP/1.1
+
+  .. sourcecode:: http
+
+     HTTP/1.1 200 OK
+     Content-Type: application/json
+
+     [{
+       id: '23',
+       title: '@foo',
+       address: '@foo',
+       type: 'twitter',
+       is_available: true,
+       provider: {
+         id: '21',
+         title: 'Twitter'
+       }
+     }]
+
+.. http:post:: /projects/(str:project_id)/channels/
+
+  Adds access to the channel with the id given in the request body for use in
+  project ``project_id``.
+
+  Only accessible if the authenticated user has :ref:`admin permission
+  <permissions-admin>`.
+
+  .. sourcecode:: http
+
+     POST /projects/21/channels/ HTTP/1.1
+
+  .. sourcecode:: http
+
+     HTTP/1.1 200 OK
+     Content-Type: application/json
+
+     {
+       "id": "23"
+     }
```

Comment:
If we specify the project under the admin `channels` section, perhaps we could allow a POST or PATCH on that endpoint to be used instead since this is only accessible to admins?